assets/js/lib/settings.js

```diff
@@ -18,6 +18,11 @@ const DEFAULT_SETTINGS = {
   editor_font_size: EDITOR_FONT_SIZE.normal,
   editor_theme: EDITOR_THEME.default,
   editor_markdown_word_wrap: true,
+  custom_section: true,
+  custom_markdown: true,
+  custom_results: true,
+  custom_output: true,
+  custom_spotlight: false,
```

Comment:
I don't think we want to store those things in Settings. Settings are long-lived and shared across notebooks. @jonatanklosko?

Reply:
@josevalim it actually makes sense, because this way we remember your last preferred view, so if you use it for a specific purpose then you don't need to reconfigure it every time.